### PR TITLE
boards/opentitan: fix verilator app support

### PR DESCRIPTION
**Pull request by twilfredo**
### Pull Request Overview

Fixes loading of the correct address layout when building an app for Verilator. Added more instructions for building and running a `libtock-c` app on Verilator.

### Testing Strategy

Compiling and running on Verilator.

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make prepush`.

### CODE REVIEW

File: boards/opentitan/README.md [hunk 1]
```diff
@@ -217,14 +217,45 @@ $ make flash-app APP=<...> OPENTITAN_TREE=/home/opentitan/
...
+$ cd libtock-c/examples/c_hello/
+$ make RISCV=1
+```
+<u>**Load and Run:**</u>
+
+Now, in the Opentitan board directory in tock (`tock/boards/opentitan/earlgrey-cw310`)
+```
+$ APP=/home/twilfred/wdc/libtock-c/examples/c_hello/build/rv32imc/rv32imc.0x20030080.0x10005000.tbf make BOARD_CONFIGURATION=sim_verilator verilator
+```
+pay attention to the flash/memory mapping of the tbf  file and ensure that it falls within the expected ranges in `layout.ld`.
```
Comment (bradjc) [inline boards/opentitan/README.md:244]: This comment doesn't seem to match the rest of this change. If someone is reading to learn how to clone the libtock-c repo and compile a libtock app, are they going to be able to check the flash and memory mapping? I suspect they wouldn't know what that means. I think either remove this line, or specify which TBF they need (it shouldn't change unless the kernel, and this readme, changes), or add some explanation.

**Review by bradjc — COMMENTED** [review]
Can you also remove the HTML from the readme?

File: boards/opentitan/README.md (hunk 1 above)
Comment (twilfredo) [reply boards/opentitan/README.md]: removed HTML (for underlines), and updated the README to specify which `tbf` to use.

**Review by alistair23 — APPROVED** [review]

**Review by bradjc — DISMISSED** [review]

**Comment by hudson-ayers** [comment]
The Netlify build has been fixed but I don't know how to force netlify to rerun other than force pushing this PR

**Comment by twilfredo** [comment]
Force pushed, re-run ci

**Review by hudson-ayers — APPROVED** [review]
bors r+

**Pull request merged**